package.json

```diff
@@ -26,7 +26,8 @@
   },
   "devDependencies": {
     "jshint": "*",
-    "mocha": "*"
+    "mocha": "*",
+    "fs-extra": "*"
```

Comment:
It's totally up to Rob - though I do think that we should stick to `yeoman-generator`'s methods. One day we'll need to update this generator to use the `generator.fs` methods anyways.